### PR TITLE
Added tests for session page.

### LINKED ISSUE #1656: Add tests for session page.
Add tests for session page.

### PR DESCRIPTION
**Pull request by agbilotia1998**
Fixes #1656 
Parent Issue #1614

### CODE REVIEW

**Comment by agbilotia1998** [comment]
@geekyd please review.

File: test/serverTest.js [hunk 1]
```diff
@@ -1131,4 +1132,39 @@ describe("Running Selenium tests on Chrome Driver", function () {
...
     });
   });
+  
+  describe('Testing Session page', function () {
+    before(function () {
+      sessionPage.init(driver);
+      sessionPage.visit('http://localhost:5000/live/preview/a@a.com/MozillaAllHands2017/sessions/session_1090.html');
+    });
+    
+    it('Get the title of the session ', function (done) {
```
Comment (geekyd) [inline test/serverTest.js]: Remove extra space.

File: test/serverTest.js [hunk 2]
```diff
@@ -1131,4 +1132,39 @@ describe("Running Selenium tests on Chrome Driver", function () {
...
+    it('Check the background color of the title', function (done) {
+      sessionPage.getSessionBackgroundColor().then(function (val) {
+        assert.equal(val, 'rgba(88, 214, 141, 1)');
+        done();
+      }).catch(function (err) {
+        done(err);
+      });
+    });
+    
+    it('Get the speaker of the session ', function (done) {
```
Comment (geekyd) [inline test/serverTest.js]: Remove extra space.

File: src/selenium/sessionPage.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+var BasePage = require('./basePage.js');
```
Comment (geekyd) [inline src/selenium/sessionPage.js]: @agbilotia1998 Good job! Lets add tests for jumping to speakers page, if the speaker image is clicked. Track if track is clicked & room when a venue is clicked.

Comment (agbilotia1998) [reply src/selenium/sessionPage.js]: Okay @geekyd .

**Comment by agbilotia1998** [comment]
@geekyd done please review!

**Comment by agbilotia1998** [comment]
@geekyd please review!

**Pull request merged**